ingestion/src/metadata/ingestion/source/atlas.py

```diff
@@ -0,0 +1,267 @@
+            bootstrap_servers.split(","),
+            metadata_config,
+        )
+        self.atlas_client = AtlasClient(config)
+        path = Path(self.config.entity_types)
+        if not path.is_file():
+            logger.error(f"File not found {self.config.entity_types}")
+            raise FileNotFoundError()
+        f = open(self.config.entity_types)
+        self.config.entity_types = json.load(f)
+        self.tables: Dict[str, Any] = {}
+        self.topics: Dict[str, Any] = {}
+
+    @classmethod
+    def create(cls, config_dict, metadata_config_dict, ctx):
+        config = AtlasSourceConfig.parse_obj(config_dict)
+        metadata_config = MetadataServerConfig.parse_obj(metadata_config_dict)
+        return cls(config, metadata_config, ctx)
+
+    def prepare(self):
+        for key in self.config.entity_types["Table"].keys():
+            self.tables[key] = self.atlas_client.list_entities(entity_type=key)
+        for key in self.config.entity_types["Topic"]:
+            self.topics[key] = self.atlas_client.list_entities(entity_type=key)
+
+    def next_record(self):
+        if self.tables:
+            for _, entity in enumerate(self.tables):
+                yield from self._parse_table_entity(entity, self.tables[entity])
+        if self.topics:
+            yield from self._parse_topic_entity()
+
+    def close(self):
+        return super().close()
+
+    def get_status(self) -> SourceStatus:
+        return self.status
+
+    def _parse_topic_entity(self):
+        for key in self.topics.keys():
+            topic_entity = self.atlas_client.get_entity(self.topics[key])
+            tpc_entities = topic_entity["entities"]
+            for tpc_entity in tpc_entities:
+                try:
+                    tpc_attrs = tpc_entity["attributes"]
+                    topic_name = tpc_attrs["name"]
+                    topic = CreateTopicRequest(
+                        name=topic_name[0:63],
+                        service=EntityReference(
+                            id=self.message_service.id, type="messagingService"
+                        ),
+                        partitions=1,
+                    )
+
+                    yield topic
+                    yield from self.ingest_lineage(tpc_entity["guid"])
+
+                except Exception as e:
+                    logger.error("error occured", e)
+                    logger.error(f"Failed to parse {topic_entity}")
+
+    def _parse_table_entity(self, name, entity):
+        for table in entity:
+            table_entity = self.atlas_client.get_entity(table)
+            tbl_entities = table_entity["entities"]
+            for tbl_entity in tbl_entities:
+                try:
+                    tbl_columns = self._parse_table_columns(
+                        table_entity, tbl_entity, name
+                    )
+                    tbl_attrs = tbl_entity["attributes"]
+                    db_entity = tbl_entity["relationshipAttributes"]["db"]
```

Comment:
Please handle relationshipAttributes and mapping from json file instead of `["db"]`